core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,47 @@
+const e2eSuperAdmin = require('../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const e2eBlogAdmin = require('../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const e2eBlogPostEditor = require('../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const testConstants = require('../puppeteer-testing-utilities/testConstants.js');
+
+const blogAdminUrl = testConstants.URLs.BlogAdmin;
+const ROLE_BLOG_ADMIN = 'BLOG_ADMIN';
+const ROLE_BLOG_POST_EDITOR = 'BLOG_POST_EDITOR';
+
+async function blogAdminUpdatingRolesAndTagsProperties() {
+  const superAdmin = await new e2eSuperAdmin();
```

Comment:
Put the "sign up as user type" stuff in the construction of the user.

Reply:
Thanks Sean/ I am currently working to pass the desired `role` as a parameter when creating a user.